package.json

```diff
@@ -8,6 +8,33 @@
     "jsonschema2md": "./cli.js"
   },
   "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./types/index.d.ts",
+        "default": "./lib/index.js"
+      }
+    },
+    "./lib/*.js": {
+      "import": {
+        "types": "./types/*.d.ts",
+        "default": "./lib/*.js"
+      }
+    },
+    "./lib/*": {
+      "import": {
+        "types": "./types/*.d.ts",
+        "default": "./lib/*.js"
+      }
+    },
+    "./lib/index": {
+      "import": {
+        "types": "./types/index.d.ts",
+        "default": "./lib/index.js"
+      }
+    },
```

Comment:
These are currently needed for backwards compatibility, in case somebody has done `import {jsonschema2md} from "@adobe/jsonschema2md/lib/index";` or `import { default as schemaLoader } from '@adobe/jsonschema2md/lib/schemaProxy';`.

However, as long as you're okay with having a **BREAKING CHANGE**, we can remove these lines.